File: app/src/androidTest/java/com/varma/hemanshu/coffeehouse/UserDetailsTest.java
package com.varma.hemanshu.coffeehouse;

import androidx.test.ext.junit.rules.ActivityScenarioRule;
import androidx.test.ext.junit.runners.AndroidJUnit4;
import androidx.test.filters.LargeTest;

import org.junit.Rule;
import org.junit.Test;
import org.junit.runner.RunWith;

import static org.junit.Assert.*;

@LargeTest
@RunWith(AndroidJUnit4.class)
public class UserDetailsTest {

    @Rule
    public ActivityScenarioRule<UserDetails> activityScenarioRule = new ActivityScenarioRule<>(UserDetails.class);

    @Test
    public void launchTest_success() {

    }

}

File: app/src/androidTest/java/com/varma/hemanshu/coffeehouse/UserDetailsTest.kt
package com.varma.hemanshu.coffeehouse

import androidx.test.ext.junit.rules.ActivityScenarioRule
import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.filters.LargeTest
import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
import org.junit.Rule
import org.junit.Test
import org.junit.runner.RunWith

@LargeTest
@RunWith(AndroidJUnit4ClassRunner::class)
class UserDetailsTest {

    @Rule
    @JvmField
    val activityScenarioRule = ActivityScenarioRule(UserDetails::class.java)

    @Test
    fun launchTest_success() {
        activityScenarioRule.scenario
    }
}